grpc-server-spring-boot-starter/src/main/java/net/devh/boot/grpc/server/autoconfigure/GrpcHealthServiceAutoConfiguration.java

```
@@ -51,8 +57,17 @@ HealthStatusManager healthStatusManager() {
 
     @Bean
     @GrpcService
+    @ConditionalOnProperty(prefix = "grpc.server", name = "health-service-type", havingValue = "GRPC",
+            matchIfMissing = true)
     BindableService grpcHealthService(final HealthStatusManager healthStatusManager) {
         return healthStatusManager.getHealthService();
     }
 
+    @Bean
+    @GrpcService
+    @ConditionalOnProperty(prefix = "grpc.server", name = "health-service-type", havingValue = "ACTUATOR")
+    @ConditionalOnBean(HealthEndpoint.class)
```

I'm thinking what happens if actuator is present, but disabled, should this then silently be ignored like it is with this annotation, or should it fail with an actual error message/missing bean exception?

---

Given that it is now an option where the default is not actuator, I will remove this conditional on the Bean. The health check would be expected; it is better to fail IMO